src/lib/cozy-client/slices/sharings.js

```diff
@@ -397,7 +397,7 @@ const getDocumentActiveSharings = (state, doctype, id) => {
       perm => perm.attributes.permissions.files.values.indexOf(id) !== -1
     ),
     ...perms.withMe.filter(
-      perm => perm.attributes.permissions.files.values.indexOf(id) !== -1
+      perm => perm.attributes.permissions.rule0.values.indexOf(id) !== -1
```

Comment:
`rule0` is a very bad name we should change.

Reply:
I cannot understand if something is wrong when I read on line 397 `perm.attributes.permissions.files.values` and 3 lines after (line 400) in the exact same scope `perm.attributes.permissions.rule0.values`.

Same appears on line 412 `p.attributes.permissions.rule0` with line 411 `p.attributes.permissions.files` and line 413 `p.attributes.permissions[type]`. It is very confusing.